### DISCUSSION TITLE
1.9版本调用rest api注册httppull源配置，结果仅创建了一个空的配置，我的参数是哪里有问题？

### DISCUSSION THREAD

**Discussion by yruipeng** [lf-edge/ekuiper#3185] [Q&A]
PUT http://localhost:9081/metadata/sources/httppull/confKeys/httppull_conf1
"httppull_conf1": {
				"url": "http://t.weather.sojson.com/api/weather/city/101030100",
				"method": "get",
				"interval": "12000",
				"timeout": "5000",
				"incremental": "false"
			}
			
			#
/
<img width="1280" alt="Snipaste_2024-09-10_15-55-55" src="https://github.com/user-attachments/assets/a127d083-e61b-4324-b1c1-83d92b33a154">

**Comment by yruipeng** [comment]
参数是放在body

**Reply by ngjaying** [reply]
{
"url": "http://t.weather.sojson.com/api/weather/city/101030100",
"method": "get",
"interval": "12000",
"timeout": "5000",
"incremental": "false"
}
